clients/cloud/rest-proxy/consume-ccsr.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+  
+# Create a consumer for Avro data, starting at the beginning of the topic's log
+docker-compose exec rest-proxy curl -X POST \
+     -H "Content-Type: application/vnd.kafka.v2+json" \
+     --data '{"name": "ci2", "format": "avro", "auto.offset.reset": "earliest"}' \
+     http://localhost:8082/consumers/cg2 | jq .
+
+# Subscribe to a topic
+docker-compose exec rest-proxy curl -X POST \
+     -H "Content-Type: application/vnd.kafka.v2+json" \
+     --data '{"topics":["test2"]}' \
+     http://localhost:8082/consumers/cg2/instances/ci2/subscription | jq .
+
+# Consume some data using the base URL in the first response.
+# Note: Issue this command twice due to https://github.com/confluentinc/kafka-rest/issues/432
```

Comment:
I assume this is still an problem? It wasn't clear reading through https://github.com/confluentinc/kafka-rest/issues/432 whether the issue still exists or not.

Reply:
@mikebin in my environment, https://github.com/confluentinc/kafka-rest/issues/432 is still an issue